tests/README.md

```diff
@@ -0,0 +1,13 @@
+# Custom WAR Packager Tests
```

Comment:
Not correct, it is only about Jenkinsfile Runner tests

Reply:
You are right.